### ISSUE TITLE
Add live demo

### ISSUE THREAD

**Issue by nmec** [ef4/physical-design-demo#6]
It would good to have a demo of this running on Heroku (or wherever really) so you don't have to download and build it to try it out.

**Comment by ef4** [comment]
Agreed, I should get it cleaned up and better documented and push it to github pages.
